### ISSUE TITLE
no cmd_vel topic

### ISSUE THREAD

**Issue by alam121** [harshmittal2210/Robotics_ws#11]
Hi ive tried the teleop package to move the robot but there are no cmd_vel topic.

```
 rosrun teleop_twist_keyboard teleop_twist_keyboard.py cmd_vel:=/atom/cmd_vel
Waiting for subscriber to connect to /atom/cmd_vel

```

```
alam@ubuntu:~/Robotics_ws$ rostopic list
/atom/camera/rgb/camera_info
/atom/camera/rgb/image_raw
/clicked_point
/clock
/gazebo/link_states
/gazebo/model_states
/gazebo/parameter_descriptions
/gazebo/parameter_updates
/gazebo/set_link_state
/gazebo/set_model_state
/initialpose
/joint_states
/move_base_simple/goal
/rosout
/rosout_agg
/scan
/tf
/tf_static
```

**Comment by harshmittal2210** [comment]
Hi 

Are you using the latest code and have installed the libraries properly?

I tried running: `roslaunch atom world.launch`

And I am getting the topic:

```
/atom/camera/parameter_descriptions
/atom/camera/parameter_updates
/atom/camera/rgb/camera_info
/atom/camera/rgb/image_raw
/atom/camera/rgb/image_raw/compressed
/atom/camera/rgb/image_raw/compressed/parameter_descriptions
/atom/camera/rgb/image_raw/compressed/parameter_updates
/atom/camera/rgb/image_raw/compressedDepth
/atom/camera/rgb/image_raw/compressedDepth/parameter_descriptions
/atom/camera/rgb/image_raw/compressedDepth/parameter_updates
/atom/camera/rgb/image_raw/theora
/atom/camera/rgb/image_raw/theora/parameter_descriptions
/atom/camera/rgb/image_raw/theora/parameter_updates
/atom/cmd_vel
/atom/odom
/clicked_point
/clock
/gazebo/link_states
/gazebo/model_states
/gazebo/parameter_descriptions
/gazebo/parameter_updates
/gazebo/performance_metrics
/gazebo/set_link_state
/gazebo/set_model_state
/initialpose
/joint_states
/move_base_simple/goal
/rosout
/rosout_agg
/scan
/tf
/tf_static
```

**Comment by alam121** [comment]
Some weird things happened but not its working.

When running teleop op command I got a warning

`the rosdep view is empty: call 'sudo rosdep init' and 'rosdep update'`

I tried installing this by:
```

alam@ubuntu:~/Robotics_ws$ rosdep update

Command 'rosdep' not found, but can be installed with:

sudo apt install python-rosdep2
```

After this my ROS melodic packages were no where to find. Roscore, roslaunch etc were not working. 
I re-install ROS melodic desktop full on the existing ROS and it installed 150MB of packages.

After this the launch file working perfectly.
